tests/realtime/redisPublisher.js

```diff
@@ -81,6 +81,29 @@ describe('redis Publisher', () => {
       })
       .catch(done);
     });
+
+    it('when tried to publish sample without aspect,'+
```

Comment:
The sample object the is finally passed to the real-time emitter should have the following fields/objects to not to break anything
1. absolutePath - as a field copied over from subject absolutePath
2. subject object always attached - having  tags field
3. aspect object always attached having tags field
I have added tests to check for these.